docs/user-guide/ze-usage.md

```diff
@@ -0,0 +1,310 @@
+# Using Zowe Explorer
+
+Use the content of this section to familiarize yourself with the extension, and make the best use of available options and features. The section contains usage tips, sample use cases for data sets, USS files, JOBs, and TSO commands.
```

Comment:
"Review this section to familiarize yourself with the extension and make the best use of available options and features." 

"...contains usage tips and sample use cases for data sets, USS files, jobs, and TSO commands."